src/containers/Root.js

```diff
@@ -6,7 +6,7 @@ export default class Root extends React.Component {
     history: React.PropTypes.object.isRequired,
     routes: React.PropTypes.element.isRequired,
     store: React.PropTypes.object.isRequired
-  }
+  };
 
```

Comment:
Yeah I can't believe they released this as a minor version without any real warning. Was triaging this all day at work, fun times :(